assets/prototype/domain/shared/dateRangeInput/formatSelect.js

```diff
@@ -0,0 +1,21 @@
+import moment from 'moment';
+
+export const FORMATS = [
+	momentFormatter('YYYY-MM-DD HH:mm:ss'),
+	{
+		formatDate: (date) => (date == null ? '' : date.toLocaleDateString()),
+		parseDate: (str) => new Date(Date.parse(str)),
+		placeholder: 'JS Date',
+	},
+	momentFormatter('MM/DD/YYYY'),
+	momentFormatter('YYYY-MM-DD'),
+	momentFormatter('YYYY-MM-DD HH:mm:ss'),
+];
```

Comment:
IMHO this should be an object with named formats (ie: keys are string constants)

because seeing something like `const format = FORMATS[FORMAT.MYSQL];`

will be more clear than seeing `const format = FORMATS[0];`

Reply:
Thx, updated: https://github.com/eventespresso/event-espresso-core/pull/1900/commits/3d4e154dce62d07bd04d8e40743ee668419af770